### PR TITLE
quickstart update

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:**
This is a patch release accounting for the quickstart yml change so that quickstart will succesfully deploy.

**This PR will result in the following new package version:**

v0.19.1

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

- Removed `hubspot__daily_ticket_history`, `hubspot__tickets` from the quickstart.yml since these models may not always be present.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [ ] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [na] The appropriate issue has been linked, tagged, and properly assigned
- [na] All necessary documentation and version upgrades have been applied
- [na] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

na

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# dbt_hubspot v0.19.1
+
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:2]: Let's add this to an under the hood section and include the PR link for this change.

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
One small change request before approval.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# dbt_hubspot v0.19.1
+[PR #149](https://github.com/fivetran/dbt_hubspot/pull/149) includes the following updates:
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:2]: This PR actually doesn't include the relevant change. Request to update this to reference the PR which made the `quickstart.yml` change.
```suggestion
[PR #148](https://github.com/fivetran/dbt_hubspot/pull/148) includes the following updates:
```

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM for release review with just one final request to change the PR reference in the CHANGELOG.

**Review by fivetran-avinash — APPROVED** [review]
@fivetran-reneeli You're good to go!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+# dbt_hubspot v0.19.1
+[PR #148](https://github.com/fivetran/dbt_hubspot/pull/148) includes the following updates:
```
[the remark this answers is not in the document]
Comment (fivetran-avinash) [reply CHANGELOG.md:2]: [This is referencing the initial PR](https://github.com/fivetran/dbt_hubspot/pull/148/files) where this change was applied, so no change needed here.

**Pull request merged**